godotdocset.py

```diff
@@ -131,23 +131,39 @@ def get_plist(name: str):
 	<string>dashtoc</string>
 </dict>
 </plist>
-'''.format(name.split('_')[0],
-		name.replace('_', ' '),
-		name.split('_')[0].lower())
+'''.format(name.split('_')[0]+version, name.replace('_', ' ')+version, name.split('_')[0].lower())
 
 
+def get_args_from():
```

Comment:
same here, please return argument parsing to main and save frompath to global variable